auto_py_to_exe/web/js/constants.js

```diff
@@ -9,31 +9,31 @@ const options_inputTypeDoubleDirectoryDest = ['datas'];
 
 const advancedSections = [
```

Comment:
Could you remove the changes from this file and create a new PR with these changes targeting `feature/153-add-chinese-translations`? I have made a new file [i18n.js](https://github.com/brentvollebregt/auto-py-to-exe/blob/feature/153-add-chinese-translations/auto_py_to_exe/web/js/i18n.js) which can hold all the translations.